src/PrestaShopBundle/Form/Admin/Sell/Product/Category/ProductCategoryType.php

```diff
@@ -44,12 +44,6 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'class' => 'category',
                 ],
             ])
-            ->add('is_default', RadioType::class, [
```

Comment:
To help rendering your collection maybe you're gonna need a "display type" Since the search entity component has been merged you can use this type:

https://github.com/jolelievre/PrestaShop/blob/071bd153a87df68c40936410bffef0a5bf3d7172/src/PrestaShopBundle/Form/Admin/Type/TextPreviewType.php